juju/juju
# Clean unused CI scripts

## Description of change
This change removes all the deprecated code from the migration to the new CI environment.

## QA steps
Nothing in CI should break :p The list should be reviewed to ensure all scripts are either no longer needed, or migrated.

The following should be migrated or deprecated:

aws.py
clean_lxd.py
clean_maas.py
clean_resources.py
gce.py
remove-manual-juju.bash
winazurearm.py

From releasetests:

agent_archive.py
azure_publish_tools.py
build_package.py
generate_agents.py
joyent-curl.bash
lp_copy_packages.py
lprelease.py
manta_sync.py
upload_packages.py
set_stream.py
